walk/config.go

```diff
@@ -135,8 +136,12 @@ func (c *Configurer) loadBazelIgnore(repoRoot string, wc *walkConfig) error {
 			log.Printf("the .bazelignore exclusion pattern must not be a glob %s", ignore)
 			continue
 		}
-		// Ensure we remove trailing slashes or the exclude matching won't work correctly
-		wc.excludes = append(wc.excludes, strings.TrimSuffix(ignore, "/"))
+
+		// Clean the path to remove any extra '.', './' etc otherwise
+		// the exclude matching won't work correctly.
+		ignore = filepath.Clean(ignore)
```

Comment:
```suggestion
		ignore = path.Clean(ignore)
```
for consistent behavior across platforms.